workspace/assets/css/vars/theme.xml

```diff
@@ -5,23 +5,126 @@
 
 		<!-- Colors -->
 		<colors>
+			<!-- Brand -->
+			<type name="symbolic">
```

Comment:
Can we comment out most of them ? If not, it will create a big ass css file...

Reply:
Yes, all unused are now commented.